### DISCUSSION TITLE
👋 Welcome PanditProject contributors!

### DISCUSSION THREAD

**Discussion by AmirSimantov** [PanditProject/Website#3] [News]
From now on, we’re using GitHub.com for communication. 

This is the place to talk about the database and the website. Your mutual help, support, and ideas are based on a voluntary subscription, of course :)

The **"Discussions"** section that you can see in the menu above is the place to:
- Ask questions you may have.
- Post answers to fellow contributors.
- Share ideas, experiences, and thoughts.
- Engage with other community members.
- Make new contributors feel welcome.

Remember that this is a community we build together!

We may use some other capabilities that GitHub offers, such as mutual documentation, teams, etc. More on this in the coming weeks; I hope 🙏

If you are new to GitHub, you will find out that the system is pretty intuitive to use. Whatever you write is open to anyone on the internet. A note for veteran GitHub users: we are currently using only "Discussions," not "Issues."

Please don't be shy and ask as many questions as you have. Please also be proactive: if you have an idea about improving PanditProject, this is the place to share them. We may implement some of the ideas according to budget, availability, cooperation, scope, and tradeoffs. The current version of PanditProject.org that we are all using is 1.0 and has been in use since March 2015. Next year, we plan to build version 2.0 that will have the same data, similar structure, and up-to-date behavior and "look & feel." So, we will delay the implementation of some of the ideas to the next version. However, we need to hear how you use the website, what you like and dislike, and how we can improve the system.

**Please comment below on what you think about this initiative.**
If you have a specific question, issue, or idea, please open a new discussion for the sake of separation of concerns.

Thank you!
Amir Simantov
PanditProject Tech Lead

**Comment by wujastyk** [comment]
Thank you, Amir, it's great to have this forum.  I look forward to help, discussion, enquiry and the growth of PanditProjects's community of users.
Dominik

**Reply by gilbh** [reply]
Likewise, congrats on the transition to GitHub and keep up to good work!
Gil
